packages/definitions/dist/fhir/r4/profiles-medplum.json

```diff
@@ -482,6 +482,16 @@
               "code" : "string"
             }]
           },
+          {
+            "id" : "Bot.runtimeVersion",
+            "path" : "Bot.runtimeVersion",
+            "definition" : "The identifier of the bot runtime environment (i.e., vmcontext, awslambda, etc).",
+            "min" : 0,
```

Comment:
Does it matter that `min` is an int, and that `max` is a string?

Reply:
Good eye.  That is how it is defined in the spec:
<img width="519" alt="image" src="https://user-images.githubusercontent.com/749094/163738600-b567f630-f469-4a03-a24e-c24b5f362e3a.png">


The common patterns are:
* `min: 0, max: "1"` (optional field)
* `min: 1, max: "1"` (required field)
* `min: 0, max: "*"` (array field)